src/vm/node_modules/VM.js

```diff
@@ -12055,16 +12067,14 @@ function deleteZone(uuid, log, callback)
 
             if (data_volumes.length === 0) {
                 log.debug({uuid: vmobj.uuid}, 'VM has no local data volumes '
-                    + 'no need to check other VMs for --volumes-from');
+                    + 'no need to check other VMs for dependant filesystems');
                 cb();
                 return;
             }
 
             log.info({uuid: vmobj.uuid, volumes: data_volumes},
                 'VM has local data volumes, checking other containers for '
-                + '--volumes-from');
-
-            // TODO can we safely limit to the same owner_uuid?
+                + 'dependant filesystems');
```

Comment:
s/dependant/dependent/g

And I mean "/g" there, I see other identical (copy/pasted?) misspellings.

Reply:
Fixed.